packages/modals/src/styled/StyledBackdrop.ts

```diff
@@ -18,21 +16,46 @@ export interface IStyledBackdropProps {
   isAnimated?: boolean;
 }
 
+const animationStyles = () => {
+  const animationName = keyframes`
+    0% {
+      opacity: 0;
+    }
+
+    to {
+      opacity: 1;
+    }
+  `;
+
+  return css`
+    animation-name: ${animationName};
+    animation-duration: 0.15s;
+    animation-timing-function: ease-in;
```

Comment:
I separated the animation CSS properties to longhand form so that an assertion can be made against `animation-duration` and `animation-timing-function` for modals that are rendered with the `isAnimated` prop. I wasn't sure if asserting against `animation-name` was suitable since it is a generated string (but maybe it might be?). Request for changes and suggestions are welcomed.